dotcom-rendering/src/components/ArticleTitle.tsx

```diff
@@ -67,30 +68,35 @@ export const ArticleTitle = ({
 	guardianBaseURL,
 	badge,
 	isMatch,
-}: Props) => (
-	<div css={[sectionStyles, badge && badgeContainer]}>
-		{badge && format.display !== ArticleDisplay.Immersive && (
-			<div css={titleBadgeWrapper}>
-				<Badge imageSrc={badge.imageSrc} href={badge.href} />
+}: Props) => {
+	const { renderingTarget } = useConfig();
+	const showBadge = renderingTarget === 'Web' && badge !== undefined;
+
+	return (
+		<div css={[sectionStyles, showBadge && badgeContainer]}>
+			{showBadge && format.display !== ArticleDisplay.Immersive && (
```

Comment:
should `format.display !== ArticleDisplay.Immersive` move into `showBadge`?

Reply:
> Is the suggestion that this is incorrect, and they should always have been checking for immersive too?

That’s what I’m not sure of… agreed that this would change the logic and would thus be best placed as a follow-up separate PR